recipe/meta.yaml

```diff
@@ -10,22 +10,23 @@ source:
   sha256: c94b40da27bb9e873566e3ccf34d4f8ab914124d41163727cc9bea91eca8f3db
 
 build:
-  number: 0
+  noarch: python
+  number: 1
 
 outputs:
   - name: {{ name }}
     build:
       noarch: python
-      script: python -m pip install . -vv
+      script: python -m pip install . -vv --no-deps --no-build-isolation
```

Comment:
This line seems to be the most important change. From the [documentation](https://pip.pypa.io/en/stable/cli/pip_install/): 

`--no-deps`: Do not install runtime dependencies (aka run requirements).

`--no-build-isolation`: Do not install build dependencies (aka host requirements) or build in a separate venv.